BUILDING.md

```diff
@@ -217,7 +217,8 @@ $ cmake ../.. -DgRPC_INSTALL=ON                \
               -DgRPC_CARES_PROVIDER=package    \
               -DgRPC_PROTOBUF_PROVIDER=package \
               -DgRPC_SSL_PROVIDER=package      \
-              -DgRPC_ZLIB_PROVIDER=package
+              -DgRPC_ZLIB_PROVIDER=package     \
+              -DgRPC_RE2_PROVIDER=package
```

Comment:
Can you move RE2 up to right below `DgRPC_PROTOBUF_PROVIDER` so that these are alphabetically ordered?

Reply:
Done, commit-id:   `2b73fcc`

Reply:
Thanks!